<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDB</class>
 <widget class="QDialog" name="CreateDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание новой базы</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/32x32/ico/add-to-database_32x32.png</normaloff>:/32x32/ico/add-to-database_32x32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Имя базы:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="baseName">
       <property name="maxLength">
        <number>64</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="typeBase">
     <property name="title">
      <string>Тип базы данных</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="emptyBase">
        <property name="text">
         <string>Чистая база</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="emptyBaseWithDirectories">
        <property name="text">
         <string>Чистая база с заполненными справочниками</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="demoBase">
        <property name="text">
         <string>База с данными для тестирования</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="log">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Создать</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/16x16/ico/apply_16x16.png</normaloff>:/16x16/ico/apply_16x16.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/16x16/ico/cancel_16x16.png</normaloff>:/16x16/ico/cancel_16x16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>baseName</tabstop>
  <tabstop>emptyBase</tabstop>
  <tabstop>emptyBaseWithDirectories</tabstop>
  <tabstop>demoBase</tabstop>
  <tabstop>createButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>CreateDB</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
